docs/components/RenameNotice/RenameNotice.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Unstyled } from '@storybook/blocks'
+import { InlineNotification } from '~components/Notification'
+
+export const RenameNotice = (): JSX.Element => {
+  return (
+    <Unstyled>
+      <InlineNotification variant="cautionary" persistent title="Deprecated">
+        {`This component will be renamed in the upcoming major release.`}
```

Comment:
Can we link out to the major releases doc for folks to find the rename value in our docs? It might preemptively nip a couple of questions in the bud for those that won't dive into our types